src/common/admin_socket.cc

```diff
@@ -507,7 +507,7 @@ class VersionHook : public AdminSocketHook {
     if (command == "0") {
       out.append(CEPH_ADMIN_SOCK_VERSION);
     } else {
-      JSONFormatter jf;
+      JSONFormatter jf(true);
```

Comment:
this practically changes the formatter to "json-pretty". my main concern would be that this could break user's tool expecting the output format of "json" formatter. so, instead using "json-pretty", probably we can just add
```c++
jf.enable_line_break();
```
for adding a line break.